WordPress/Classes/Extensions/NSFetchedResultsController+Helpers.swift

```diff
@@ -3,6 +3,21 @@ import Foundation
 
 extension NSFetchedResultsController
 {
+    /// Returns whether an indexPath represents the last row in it's section, or not
+    ///
+    func isLastRowInSection(indexPath: NSIndexPath) -> Bool {
```

Comment:
Hmm okay you do use the row below. Maybe `forIndexPath` for the parameter? Maybe I'm just being stupid. 😄